bin/init.bash

```diff
@@ -212,6 +218,10 @@ set_object_storage() {
     esac
 
     replace_set_me "${file}" ".objectStorage.type" "\"${object_storage_type}\""
+    if [ "$object_storage_type" == "azure" ]; then
+        replace_set_me "${file}" ".objectStorage.azure.resourceGroup" "${CK8S_ENVIRONMENT_NAME}"
```

Comment:
I think we should distinguish this resource group from the other resource groups by suffixing it.

```suggestion
        replace_set_me "${file}" ".objectStorage.azure.resourceGroup" "${CK8S_ENVIRONMENT_NAME}-storage"
```